src/Microsoft.ML.Core/Data/IDataView.cs

```diff
@@ -146,43 +136,50 @@ public interface IRowCursorConsolidator
     public abstract class Row : IDisposable
     {
         /// <summary>
-        /// This is incremented when the underlying contents changes, giving clients a way to detect change.
-        /// Generally it's -1 when the object is in an invalid state. In particular, for an <see cref="RowCursor"/>, this is -1
-        /// when the <see cref="RowCursor.State"/> is <see cref="CursorState.NotStarted"/> or <see cref="CursorState.Done"/>.
+        /// This is incremented when the underlying contents changes, giving clients a way to detect change. Generally
+        /// it's -1 when the object is in an invalid state. In particular, for an <see cref="RowCursor"/>, this is -1
+        /// when the <see cref="RowCursor.State"/> is <see cref="CursorState.NotStarted"/> or <see
+        /// cref="CursorState.Done"/>.
         ///
-        /// Note that this position is not position within the underlying data, but position of this cursor only.
-        /// If one, for example, opened a set of parallel streaming cursors, or a shuffled cursor, each such cursor's
-        /// first valid entry would always have position 0.
+        /// Note that this position is not position within the underlying data, but position of this cursor only. If
+        /// one, for example, opened a set of parallel streaming cursors, or a shuffled cursor, each such cursor's first
+        /// valid entry would always have position 0.
         /// </summary>
         public abstract long Position { get; }
 
         /// <summary>
-        /// This provides a means for reconciling multiple streams of counted things. Generally, in each stream,
-        /// batch numbers should be non-decreasing. Furthermore, any given batch number should only appear in one
-        /// of the streams. Order is determined by batch number. The reconciler ensures that each stream (that is
-        /// still active) has at least one item available, then takes the item with the smallest batch number.
+        /// This provides a means for reconciling multiple rows that have been produced generally from
+        /// <see cref="IDataView.GetRowCursorSet(Func{int, bool}, int, Random)"/>. When getting a set, there is a need
+        /// to, while allowing parallel processing to proceed, always have an aim thatthe original order should be
+        /// reconverable. So: for any cursor implementation, batch numbers should be non-decreasing. Furthermore, any
```

Comment:
recoverable?

Reply:
Are you saying that the original order should always be maintained?

Reply:
Given a cursor set, you should be (in principle) able to recover the order that would have been returned from a similarly configured call to the serial `GetRowCursor` by exploiting the `Batch` property, yes. I will perhaps try to make this more clear in the docs.

Reply:
Whether I succeeded in clarifying though is another matter, if you could read over it and see if my few additional explanatory sentences help or hurt, I'd appreciate it, thanks @singlis !